Makefile.am

```diff
@@ -110,6 +110,7 @@ libnlinclude_netlink_route_HEADERS = \
 	include/netlink/route/neightbl.h \
 	include/netlink/route/netconf.h \
 	include/netlink/route/nexthop.h \
+	include/netlink/route/nh.h \
```

Comment:
I added a script to automatically format new source files with `clang-format`. There is also a github action that tests the format, but it was broken. If you would rebase to latest `main`, you would see that the github test `clang-format` fails due to that.

If you would, please reformat the files. As the tool would tell you:
```
Fix it by running `"./tools/clang-format.sh"` using clang-format version 16.0.6 (Fedora 16.0.6-2.fc38)
Alternatively, run `./tools/clang-format-container.sh` to use a podman container named "libnl-code-format-f38".
```

---

If that is cumbersome for you (e.g. because you don't use Fedora 38 with the right clang-format version and don't have podman working), you can also ignore that, then I will reformat your patch before merging. Just ignore it.

Reply:
after a couple of tries it worked with my clang-format-14